src/index.ts

```diff
@@ -1 +1,18 @@
-console.log('Beuty code comming to this proqject!');
\ No newline at end of file
+const express = require("express");
+import {Express, Router} from "express";
+
+const router = require("./routes/routes");
+const participantsRouter = require("./routes/participants.route");
+const meetingRouter = require("./routes/meetings.route");
+const roomsRouter = require("./routes/rooms.route");
+
+const app : Express  = express();
+
+app.use("/", router);
```

Comment:
as vrea altcela la rute ... incearca sa definesti rutele in de exemplu cum ai rooms.route.ts ...si aici vi si spui doar 
`app.use(roomsRoutes);`

Reply:
Rezolvat! :)